supportingcharts/README.md

```diff
@@ -64,94 +65,132 @@ spec:
   entityOperator:
     topicOperator: {}
     userOperator: {}
-````
- 
-Which creates a cluster by using the following command
-````
+```
+
+Use this file with the following command to create the Kafka cluster:
+
+```bash
 oc apply -f cluster.yaml -n lightbend
-````
-You can also get information about clusters by running the following command
-````
+```
+
+You can also get information about any installed clusters under the `lightbend` namespace with the following command:
+
+```bash
 oc get kafka -n lightbend
-````
-Programmatic access to cluster (inside openshift cluster) by client in this case is through the service `sample-cluster-kafka-brokers` using host `sample-cluster-kafka-brokers.lightbend.svc`  and port `9092`
-
-## Cassandra installation
-     
-Cassandra installation is based on the following [helm chart](../../supportingcharts/extendedcassandrachart). For more information on this chart refer to documentation here. Cassandra-reaper requires privilege user (yes its a security risk, but okay for sample application). This can be achieved using the following command 
-````     
-oc adm policy add-scc-to-user anyuid -nsample -z default
-````
-After this command is executed - install chart
-````     
-helm install kubernetes/charts/extendedcassandrachart/
-````     
+```
+
+Programmatic access to the Kafka cluster inside the OpenShift or Kubernetes cluster is through the service `sample-cluster-kafka-brokers` using host `sample-cluster-kafka-brokers.lightbend.svc`  and port `9092`
+
```

Comment:
Assuming installation project is lightbend

Reply:
Added sentence about the namespace